src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+tree.addEventListener('click', (e) => {
+  const closestList = e.target.closest('li');
+
+  if (closestList.children.length !== 0) {
+    const listStyle = closestList.children[0].style;
+
+    if (listStyle.display !== 'none') {
+      listStyle.display = 'none';
+    } else {
+      listStyle.display = 'block';
```

Comment:
Toggling the display style directly can lead to issues if there are other stylesheets or scripts that also affect the display of these elements. A more robust solution would be to toggle a class on the element, and define the display behavior using CSS.